scout/server/blueprints/cases/controllers.py

```diff
@@ -230,24 +270,7 @@ def case(store, institute_obj, case_obj):
             hpo_term["phenotype_id"]
         )
 
-    if case_obj.get("rank_model_version"):
-        rank_model_link = "".join(
-            [
-                current_app.config.get("RANK_MODEL_LINK_PREFIX", ""),
-                str(case_obj["rank_model_version"]),
-                current_app.config.get("RANK_MODEL_LINK_POSTFIX", ""),
-            ]
-        )
-        case_obj["rank_model_link"] = rank_model_link
-
-    if case_obj.get("sv_rank_model_version"):
-        case_obj["sv_rank_model_link"] = "".join(
-            [
-                current_app.config.get("SV_RANK_MODEL_LINK_PREFIX", ""),
-                str(case_obj["sv_rank_model_version"]),
-                current_app.config.get("SV_RANK_MODEL_LINK_POSTFIX", ""),
-            ]
-        )
+    _link_rank_models(case_obj)
```

Comment:
```suggestion
    _set_rank_models(case_obj)
```

Reply:
This is an interesting intermediate function that could fit as a Case object internal method, but partly does the controllers job. I guess it could be split into part of each. 😸